src/symbol-icon-theme.json

```
@@ -273,6 +275,8 @@
     "css": "brackets-sky",
     "test": "code-orange",
     "js": "js",
+    "mjs": "mjs",
+    "cjs": "cjs",
```

I don't think you need to re-define additional variants, you can just do:

```suggestion
    "mjs": "js",
    "cjs": "js",
```